src/pages/blog/starlingx-release-4.md

```diff
@@ -0,0 +1,35 @@
+---
+templateKey: blog-post
+title: StarlingX 4.0 is here!
+author: Ildiko Vancsa
+date: 2020-08-03T01:32:05.627Z
+category: 
+  - label: News & Announcements
+    id: category-A7fnZYrE1
+---
+
+The StarlingX community has just launched the fourth release of the project with various new features and enhancements to the platform.<!-- more -->
+
+The project is approaching a half-year release cycle and this article will give you some insights about what the contributors were working on during the first half of this year.
+
+As the StarlingX platform is fine tuned for edge computing and IoT use cases there are several requirements it has to fulfill in areas such as performance, robustness and security which were in focus during the 4.0 release cycle as well. This blog post will give you insights to some of the new features and components that got integrated into the platform.
```

Comment:
The StarlingX platform is fine tuned for edge computing and IoT use cases and the contributors focused on requirements for performance, robustness and security goals during the 4.0 release cycle. This blog post will give you insights on some of the new features and components that were integrated into the platform.